analytics/ls-aws-sqs-default/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM docker.elastic.co/logstash/logstash:6.5.1
```

Comment:
Please change to using logstash-oss
https://github.com/Bit-Clouded/Angostura/blob/8dc75b5b7fcfbc7e0d29454140afaf9b79ef5f5c/analytics/ls-aws-kinesis/Dockerfile#L1